infinitude

```diff
@@ -317,8 +317,8 @@ post '/systems/:system_id/status' => sub {
 
 	$changes = 'true' unless $store->get('systems.xml'); #Force a change cycle if we have no stat config
 	my $xml = XML::Simple::Minded->new({
-		serverStatus => {
-			 version => '1.6',
+		status => {
```

Comment:
Isn't this determined by the version, though? 🤔 
I guess we may as well track the most recent version where possible, since people tend to slam upgrade buttons when presented with them.